Installation issue

On a fresh `Ubuntu 14.04` install from from digital ocean 
After running the following commands:

    apt-get install make git ruby -y
    git clone git@github.com:project-nsmg/figure.git
    cd figure
    make install
    
I got this error:

    root@figure:~/figure# make install
    wget -qO /usr/local/bin/sshcommand https://raw.github.com/progrium/sshcommand/master/sshcommand
    chmod +x /usr/local/bin/sshcommand
    sshcommand create figure /usr/local/bin/figure
    Adding user `figure' ...
    Adding new group `figure' (1000) ...
    Adding new user `figure' (1000) with group `figure' ...
    Creating home directory `/home/figure' ...
    Copying files from `/etc/skel' ...
    wget -qO /tmp/pluginhook_0.1.0_amd64.deb https://s3.amazonaws.com/progrium-pluginhook/pluginhook_0.1.0_amd64.deb
    dpkg -i /tmp/pluginhook_0.1.0_amd64.deb
    Selecting previously unselected package pluginhook.
    (Reading database ... 88502 files and directories currently installed.)
    Preparing to unpack .../tmp/pluginhook_0.1.0_amd64.deb ...
    Unpacking pluginhook (0.1.0) ...
    Setting up pluginhook (0.1.0) ...
    lsmod | grep aufs || modprobe aufs || apt-get install -qq -y linux-image-extra-`uname -r` > /dev/null
    apt-get install -qq -y curl
    egrep -i "^docker" /etc/group || groupadd docker
    usermod -aG docker figure
    curl --silent https://get.docker.io/gpg | apt-key add -
    gpg: no valid OpenPGP data found.
    make: *** [docker] Error 2
    root@figure:~/figure# 
    
Any thoughts?

--- a/Makefile
+++ b/Makefile
@@ -75,8 +75,8 @@ docker: aufs
 	apt-get install -qq -y curl
 	egrep -i "^docker" /etc/group || groupadd docker
 	usermod -aG docker figure
-	curl --silent https://get.docker.io/gpg | apt-key add -
-	echo deb http://get.docker.io/ubuntu docker main > /etc/apt/sources.list.d/docker.list
+	curl --silent --location https://get.docker.com/gpg | apt-key add -
+	echo deb https://get.docker.com/ubuntu docker main > /etc/apt/sources.list.d/docker.list
 	apt-get update
 ifdef DOCKER_VERSION
 	apt-get install -qq -y lxc-docker-${DOCKER_VERSION}

